Example/SnapshotTests/BPKSnapshotTest.h

```diff
@@ -22,7 +22,7 @@
         int correctMinorVersion = 0;                                                                                                                 \
         NSString *deviceName = [[UIDevice currentDevice] name];                                                                                      \
         NSOperatingSystemVersion deviceOSVersion = [[NSProcessInfo processInfo] operatingSystemVersion];                                             \
-        BOOL validDevice = [deviceName isEqual:correctDeviceName];                                                                                   \
```

Comment:
Is this change unrelated then?

Reply:
Nah - because when running concurrently the device name might be `iPhone 8` or `Clone 1 of iPhone 8` or `Clone 2 of iPhone 8` etc